lib/dirty/dirty.js

```diff
@@ -5,205 +5,199 @@ var fs = require('fs'),
+            this._data.set(row.key, row.val);
+          }
+          return '';
+        });
+      })
+      .on('end', () => {
+        if (buffer.length) {
+          this.emit('error', new Error('Corrupted row at the end of the db: ' + buffer));
         }
-        return '';
+        this.emit('load', this._data.size);
+      })
+      .on('close', () => {
+        this._readStream = null;
+        this.emit('read_close');
       });
-    })
-    .on('end', () => {
-      if (buffer.length) {
-        this.emit('error', new Error('Corrupted row at the end of the db: '+buffer));
+
+    this._writeStream = fs.createWriteStream(this.path, {
+      encoding: 'utf-8',
+      flags: 'a'
+    });
+
+    this._writeStream.on('drain', () => {
+      this._waitForDrain = false;
+      if (!this._queue.size) {
+        if (this._inFlightWrites <= 0) this.emit('drain');
+      } else {
+        this._flush();
       }
-      this.emit('load', this._data.size);
-    })
-    .on('close', () => {
-      this._readStream = null;
-      this.emit('read_close');
     });
 
-  this._writeStream = fs.createWriteStream(this.path, {
-    encoding: 'utf-8',
-    flags: 'a'
-  });
+    this._writeStream.on('close', () => {
+      this._writeStream = null;
+      this.emit('write_close');
+    });
+  }
 
-  this._writeStream.on('drain', () => {
-    this._waitForDrain = false;
-    if (!this._queue.size) {
-      if (this._inFlightWrites <= 0) this.emit('drain');
-    } else {
-      this._flush();
+  _flush() {
+    if (!this._queue.size || this._waitForDrain) return;
+    this._writeStream.cork();
+    for (const [key, cbs] of this._queue) {
+      this._queue.delete(key);
+      const data = JSON.stringify({ key: key, val: this._data.get(key) }) + '\n';
+      ++this._inFlightWrites;
+      this._waitForDrain = !this._writeStream.write(data, (err) => {
+        if (!cbs.length && err != null) this.emit('error', err);
+        if (--this._inFlightWrites <= 0 && !this._waitForDrain) this.emit('drain');
+        for (const cb of cbs) cb(err);
+      });
+      if (this._waitForDrain) break;
     }
-  });
-
-  this._writeStream.on('close', () => {
-    this._writeStream = null;
-    this.emit('write_close');
-  });
-};
-
-Dirty.prototype._flush = function() {
-  if (!this._queue.size || this._waitForDrain) return;
-  this._writeStream.cork();
-  for (const [key, cbs] of this._queue) {
-    this._queue.delete(key);
-    const data = JSON.stringify({key: key, val: this._data.get(key)})+'\n';
-    ++this._inFlightWrites;
-    this._waitForDrain = !this._writeStream.write(data, (err) => {
-      if (!cbs.length && err != null) this.emit('error', err);
-      if (--this._inFlightWrites <= 0 && !this._waitForDrain) this.emit('drain');
-      for (const cb of cbs) cb(err);
-    });
-    if (this._waitForDrain) break;
+    this._writeStream.uncork();
   }
-  this._writeStream.uncork();
-};
+}
+
+Dirty.Dirty = Dirty;
+module.exports = Dirty;
+exports.Dirty = Dirty;
```

Comment:
This line has no effect because of the line above it. It is more idiomatic to do this:

```javascript
module.exports = exports = Dirty;
```